### ISSUE TITLE
Performance

### ISSUE THREAD

**Issue by talaikis** [OsamaAbbas/bytenode#27]
With API server having 100k lines of code (2.89 MB), compiled jsc - 4.36 MB.

With bytenode:

```bash
avg rps: 14443.6
avg. latency: 0.22 ms
```

Without bytenode:

```bash
avg rps:  17303.64
avg latency: 0.08 ms
```

**Comment by OsamaAbbas** [comment]
This maybe due to Bytenode itself, or the internals of v8 or node.

If it is Bytenode's fault, I guess it maybe because of this issue: https://github.com/OsamaAbbas/bytenode/pull/7 . I will do some tests and get back to you.

**Comment by OsamaAbbas** [comment]
I made some huge `.js` files. However, there was some sort of repetition so I could not reproduce your problem (the `.jsc` files were extremely small and no big difference in performance). Also, I don't know how did you do the benchmark, so I used the simple `time` command.

So now we have two options:

Option A: You manually change the lines 88 and 175 in `bytenode/index.js` file, so that the dummyCode uses a plain space, like this:

`dummyCode = '"' + " ".repeat(length - 2) + '"'; `

and repeat your tests. If you get the same lag we would know that the problem is NOT in bytenode itself.

Option B: I can do the tests if you know an open-source file that is as huge as your file, or at least you give me something I could use because I can't write 100k lines of code without repetition.
